content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+Whether you’re a LocalStack user or just getting started on your cloud development journey, we have a lot of exciting features to share with you! With LocalStack, you can now:
+
+- Test disaster recovery simulations such as region failover, DNS failover, and service failures.
+- Explore and progressively tighten security around IAM policies as your application takes shape.
+- Create ephemeral cloud sandboxes for testing your infrastructure locally or on CI pipelines.
+- Activate analytics for LocalStack in CI runs to get insights into your infrastructure test runs.
+- Utilize enhanced, faster AWS services such as S3, Step Functions, DynamoDB and ElastiCache.
+- Get started with a fully-local developer environment with the new LocalStack Desktop Application.
+
+And much more! LocalStack makes it easy to build & test local-first cloud applications, all while improving the **Inner Dev Loop**. We are excited to share our recent developments in this release, and discuss how you can get started with them!
+
+## What’s new in LocalStack 3.0?
+
+In previous releases, we supplemented our cloud emulation capabilities with parity, performance, and interoperability. In LocalStack 3.0, our core areas of focus has been — **resilience**, **efficiency**, and **flexibility**. With LocalStack, we aim to power your local development productivity and cover the most sophisticated enterprise use cases. Collectively, the new features in the release won't just cut away your slow *code → deploy → test → redeploy → ...* loops, but also help your team to confidently ship faster and more secure cloud applications. You can explore *all* the new features by upgrading your LocalStack setup or downloading LocalStack 3.0 today! [Get your free LocalStack account](https://app.localstack.cloud/sign-up).
+
+Let's take a look at our latest release's new features and enhancements to see how you can get the most out of them!
+
+* [New S3 provider](#new-s3-provider)
+* [New Step Functions provider](#new-step-functions-provider)
+* [New ElastiCache provider](#new-elasticache-provider)
+* [New features for Chaos Engineering](#new-features-for-chaos-engineering)
+* [IAM Policy Stream on Web Application](#iam-policy-stream-on-web-application)
+* [Ephemeral Environments for LocalStack](#ephemeral-environments-for-localstack)
+* [CI Analytics for LocalStack](#ci-analytics-for-localstack)
+* [All-new LocalStack Desktop Application](#all-new-localstack-desktop-application)
+* [Multi-region and Multi-account support](#multi-region-and-multi-account-support)
+* [LocalStack Networking initiative](#localstack-networking-initiative)
+* [Miscellaneous](#miscellaneous)
+
```

Comment:
like the list!

Reply:
for the next releases, as this list grows we can group items together (aws services, web app, networking/setup, ...)